dspace-api/src/main/java/org/dspace/storage/bitstore/AzureBlobStorageBitStoreService.java

```diff
@@ -0,0 +1,395 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.storage.bitstore;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.dspace.content.Bitstream;
+import org.dspace.core.Utils;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.springframework.beans.factory.annotation.Required;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.BlobProperties;
+import com.microsoft.azure.storage.blob.BlockEntry;
+import com.microsoft.azure.storage.blob.CloudBlob;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+
+/**
+ * @author Ernani Joppert Pontes Martins
+ *
+ */
+public class AzureBlobStorageBitStoreService implements BitStoreService {
+
+	/** log4j log */
+	private static Logger log = Logger.getLogger(AzureBlobStorageBitStoreService.class);
+
+	private final String containerReferenceDefaultPrefix = "container-dspace";
```

Comment:
Is this the name of the location in Azure?  If so, I wonder if this should be configurable (from dspace.cfg).

Reply:
This is just a default prefix if the user forgets to set it up in dspace.cfg or wants to use the default prefix set here.